plugins/GSdx/Renderers/Common/GSVertexTrace.cpp

```diff
@@ -601,4 +601,19 @@ void GSVertexTrace::CorrectDepthTrace(const void* vertex, int count)
 	} else {
 		m_eq.z = 0;
 	}
+
+	/* FIXME seems like this code could be much easier/efficient
+	const GSVertex* RESTRICT v = (GSVertex*)vertex;
+	uint32 z = v[0].XYZ.Z;
+
+	for (int i = 1; i < count; i++) {
+		if (z !=  v[i].XYZ.Z) {
+			m_eq.z = 0;
+			return;
+		}
+	}
+	m_eq.z = 1;
+
+
+	*/
```

Comment:
Fixme note/code also isn't needed, there are many improvements that can be made ofc but let's keep the focus of the main goal of the pr.